application/src/main/java/org/thingsboard/server/config/TbRuleEngineSecurityConfiguration.java

```diff
@@ -40,9 +40,10 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                         .frameOptions(config -> {}).disable())
                 .cors(cors -> {})
                 .csrf(AbstractHttpConfigurer::disable)
-                .authorizeRequests()
-                .requestMatchers("/actuator/prometheus").permitAll()
-                .anyRequest().authenticated();
+                .authorizeHttpRequests(config -> config
+                        .requestMatchers("/actuator/prometheus").permitAll())
+                .authorizeHttpRequests(config -> config
+                        .anyRequest().authenticated());
```

Comment:
```suggestion
                .authorizeHttpRequests(config -> config
                        .requestMatchers("/actuator/prometheus").permitAll()
                        .anyRequest().authenticated());
```

Reply:
it works as you suggested

![image](https://github.com/thingsboard/thingsboard/assets/79898499/eea6729b-a531-4295-827c-4f96678fc840)

others is 403

![image](https://github.com/thingsboard/thingsboard/assets/79898499/008acb94-76ee-448d-ae2c-486a46e7cd2e)